### PR TITLE
AWS setup

### PR DESCRIPTION
**Pull request by benfurber**
Mainly the dotenv stuff linking to DB.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -17,12 +18,14 @@
     "smtp",
     "server"
   ],
-  "author": "Igor Ryabchuk, George Sykes, Ben Furber, John Newman, Charlene Chetcuti",
+  "author": "",
```
Comment (nixlim) [inline package.json]: What is the rationale for moving this to contributors?

File: package.json [hunk 2]
```diff
@@ -41,5 +44,15 @@
...
     ]
-  }
+  },
+  "engines": {
+    "node": "8.11.1"
+  },
+  "contributors": [
+    "Charlene Chetcuti",
+    "Ben Furber",
+    "John Newman",
```
Comment (JohnNewman1) [inline package.json]: Can i be moved up please

File: package.json (hunk 1 above)
Comment (benfurber) [reply package.json]: AWS was very strict about the structure of this file.

File: package.json (hunk 2 above)
Comment (benfurber) [reply package.json]: Sure, if you change your surname.

**Pull request merged**